index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+     <div class="welcome">
+        <h1>Hi there</h1>
+     </div>
+     <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+     </div>
```

Comment:
- [ ] The hamburger menu icon is wrapped with a `nav` tag. I suggest you use an icon instead of the three `divs`.  The `nav` tag is for semantics reason and the right position for convention and user experience for mobile devices.